<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLabel" name="labelSecondTable">
      <property name="text">
       <string>Вероятности переходов по второй случайной величине:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableWidget" name="secondTransitionTable"/>
    </item>
    <item row="10" column="0">
     <widget class="QSpinBox" name="stateSpinBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="firstTransitionTable"/>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelLambds">
      <property name="text">
       <string>Значения случайного процесса и параметры λ для случайной величины:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="lambdaInput"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelCountStates">
      <property name="text">
       <string>Количество состояний:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelFirstTable">
      <property name="text">
       <string>Вероятности переходов по первой случайной величине:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="alphaInput"/>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="timeSpinBox"/>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelTime">
      <property name="text">
       <string>Время работы:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelAplpha">
      <property name="text">
       <string>Параметры α для случайной величины:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
